### ISSUE TITLE
No longer working on firefox

### ISSUE THREAD

**Issue by RedSparr0w** [RedSparr0w/IndieGala-Helper#6]
Extention no longer working correctly on >= firefox v50

Page | Status
:---: | :---: 
Giveaways | ✖️ Not Working
Profile | ✔️ Working
Bundles | ✔️ Working

**Comment by Revadike** [comment]
What exactly is not working?

**Comment by RedSparr0w** [comment]
Pretty much everything last i checked xD,
Haven't tried the latest version yet though.

Will give it a try tomorrow and see what's still broken for firefox.

**Comment by Revadike** [comment]
Are you using tampermonkey on firefox? I often experience firefox has pretty much issues with all other userscript managers.

**Comment by RedSparr0w** [comment]
Seems to be working on the newest release of FF.
Did minimal testing though, so hopefully no bugs.

Edit: Only thing that needed changing was the addresses for ajax calls had to be absolute and not relative.
old: `/whateverqueryhere`
new: `https://www.indiegala.com/whateverqueryhere`

**Comment by Lucki** [comment]
Well, in general it works for me here, but most of the time the options are not visible. It's pretty much random if the options are available. Most of the time they are all disabled, no steamID is set and the blacklist is not clickable.
If the options panel was available once the helper works as expected, even if the options aren't shown again.

I've tried to get more information with the web console (ctrl+shift+k), but when I have them open, the options panel works **every** time. Closing the web console gets back to the old random behavior.

<img src="https://user-images.githubusercontent.com/1408843/30297075-c4d4a248-9746-11e7-8b73-92ad5fc2cd16.png" width="300">
